SU-SWS/stanford_profile
# D8CORE-2074: Fixing the line height in the tables to em

# READY FOR REVIEW

# Summary
- changing line height to em from rem

# Needed By (Date)
- 5/26

# Urgency
- Medium

# Steps to Test

1. Pull in this branch.
2. Create a table
3. Verify multiple rows aren't overlaying each other.

# Affected Projects or Products
- It is part of the WYSIWYG

# Associated Issues and/or People
- D8CORE-2074
- @cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
